src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
move repeatable colors in the variables

Reply:
did not understand what you mean. can you give an example?

Reply:
You can create variables for repeatable values. Here is the example 
```
:root {
  --main-color: #00acdc;
}

.nav-link:hover {
  color: var(--main-color);
}

Reply:
Hi, i checked this

Reply:
but I still don't understand why to use this^
:root {
  --main-color: #00acdc;
}

.nav-link:hover {
  color: var(--main-color);
}

If after applying your option, we lose the shadow when clicking